stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -3272,7 +3333,8 @@ final internal class _Native${Self}BridgingStorage<${TypeParametersDecl}>
     BridgedNativeStorage.Storage? {
     get {
       if let ref = _stdlib_atomicLoadARCRef(object: _heapBufferBridgedPtr) {
-        return unsafeDowncast(ref, to: BridgedNativeStorage.Storage.self)
+        return BridgedNativeStorage.Storage(_buffer: unsafeDowncast(ref, 
```

Comment:
Formatting rule for the standard library: "if you have an open delimiter on a line and its closing delimiter doesn't fit on the same line, break the line after the open delimiter."  That means we break after the first parenthesis above.

I haven't been looking for violations of this rule elsewhere, so you should do an audit.